boba-chain-ops/Makefile

```diff
@@ -1,33 +1,33 @@
 test:
-	go test ./...
+	go test -tags nosilkworm ./...
```

Comment:
Nit: Might be better to declare a GOTAGS variable and re-use it in case we want to add more down the line

Reply:
Added!